src/Angor/Client/Pages/Invest.razor

```diff
@@ -1032,10 +1032,11 @@ else
     private async Task SaveInvestmentsListToNostrAsync()
     {
         var words = await passwordComponent.GetWalletAsync();
-        var rootNostrPrivateKey = await _derivationOperations.DeriveProjectNostrPrivateKeyAsync(words, 0);
-        var rootNostrPrivateKeyHex = Encoders.Hex.EncodeData(rootNostrPrivateKey.ToBytes());
+        var storageAccountKey = _derivationOperations.DeriveNostrStoragePubKeyHex(words);
```

Comment:
It would help to add the PubKey in the name so it'a clear

Reply:
Yeah but I hid this complexity in the method on purpose, better no?